docs/navigation.html

```diff
@@ -0,0 +1,460 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Navbars</title>
+    <link rel="stylesheet" href="../components/navbar/navbar.css">
+    <meta name="theme-color" content="#2C2929">
+
```

Comment:
Question: I am not sure about this content

Reply:
> Question: I am not sure about this content

This meta tag add color to the url header on mobile devices.
![Screenshot_20220220-201457](https://user-images.githubusercontent.com/36101139/154860039-813b86dd-158c-4f1b-852c-865df5f4fa82.png)